provisioners/puppet/manifests/author.pp

```diff
@@ -62,11 +64,55 @@
     jvm_opts       => '-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps -XX:+PrintTenuringDistribution -XX:+PrintGCApplicationStoppedTime -XX:+HeapDumpOnOutOfMemoryError',
   }
 
+  file { '/etc/puppetlabs/puppet/aem.yaml':
+    ensure  => file,
+    content => epp('/tmp/templates/aem.yaml.epp', {'port' => $aem_port}),
+    mode    => '0644',
+    owner   => 'root',
+    group   => 'root',
+  }
+
+  # Confirm AEM Starts up and the login page is ready.
+  aem_aem { 'Wait until login page is ready':
+    ensure  => login_page_is_ready,
+    require => [Aem::Instance['aem'], File['/etc/puppetlabs/puppet/aem.yaml']],
+  }
+
+  file { '/tmp/aem-healthcheck-content':
+    ensure => directory,
+    mode   => '0775',
+    owner  => $packer_user,
+    group  => $packer_group,
+  } ->
+  wget::fetch { 'https://github.com/shinesolutions/aem-healthcheck/releases/download/v1.2/aem-healthcheck-content-1.2.zip':
```

Comment:
Just like aem quickstart source, this needs to be configurable as well.
The alternative source is http://central.maven.org/maven2/com/shinesolutions/aem-healthcheck-content/1.2/aem-healthcheck-content-1.2.zip